### DISCUSSION TITLE
Looking for EUDB compliance support.

### DISCUSSION THREAD

**Discussion by klennstrom** [microsoft/cpp_client_telemetry#1040] [General]
We are a library that is used by several apps to route telemetry to aria. We are needing for our library and the data it sends to be EUDB compliant. I was not able to find any examples in the included docs which identify what exactly we need to do in order to ensure compliance. Is there a resource somewhere that makes this process simple? The 1ds cpp interface is quite wide, and I don't want to guess here.

**Comment by lalitb** [comment]
1DS SDK doesn't do any automatic routing, the EuDB compliant endpoint URL is supposed to be available during SDK startup.

**Comment by maxgolov** [comment]
https://github.com/microsoft/cpp_client_telemetry/pull/1191
